### ISSUE TITLE
Move prettier config files to the root and remove settings.json

### ISSUE THREAD

**Issue by to-sta** [activist-org/activist#363]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The VScode extension for prettier is looking for the configuration files in the root of the project. Currently the files are in the `./frontend` and because of that settings.json was added. 

settings.json
```json
{
    "prettier.configPath": "frontend/.prettierrc",
    "prettier.ignorePath": "frontend/.prettierignore"
}
```

Task is to move the prettier configuration files to the root and remove the settings.json file enterly. Than make sure everything for still works properly. 

Test:

- CLI
- Extension
- Workflow

**Comment by to-sta** [comment]
@andrewtavis i had a look into this topic. I encountered some issues when moving the prettier config files to the root of the project. The VScode extension will work but then we need to use a prefix for `yarn` or `npm` to run `prettier` outside of the directory where `node_modules` and `package.json` are.

After looking at other large open source projects, i think it is common practice to have all configuration files in the root directory (e.g. mypy.ini, requirements.txt, tsconfig.json, package.json,...). Then we do not have to use a prefixes.

Currently i see two options:

1. Move prettier config. files to the root, delete settings.json and add a prefix to the frontend workflow (then the prettier cli will not work, might work with adding args that specify the path to the config files tho)
2. Move all configuration files to the root directory

What you think about moving all configuration files to the root directory?

**Comment by andrewtavis** [comment]
Initial thoughts on moving all the config files to the root are a bit apprehensive, @to-sta. If we can at least have the workflow to check if formatting’s correct still working then I’d prefer to have it all compartmentalized. We can reconsider when we move to micro services and maybe have the frontend as its own repo at that point. As of now we can just ask the contributor to format files that need it in another commit or do it ourselves :)

Does this sound ok? I just feel that having the structure that we have right now will help make things more clear. Plus we’d likely need to make a bunch of changes to deployment of the config files are moving.

**Comment by to-sta** [comment]
@andrewtavis yeah i am good with both options 😃. Just thought i should mention this. Will make a PR with first option incl. the frontend workflow.
